app/src/main/res/values/themes.xml

```diff
@@ -82,6 +84,20 @@
         <item name="android:colorBackground">@color/backgroundDuskDawn</item>
         <item name="actionBarTintColor">@color/appBarTextDuskDawn</item>
     </style>
+    <style name="Theme.Tachiyomi.FlatLime" parent="Theme.Tachiyomi">
+        <item name="colorPrimary">@color/backgroundLime</item>
+        <item name="colorPrimaryDark">@color/backgroundLime</item>
+        <item name="colorAccent">@color/colorAccentLime</item>
+        <item name="colorAccentText">@color/colorAccentLime</item>
+        <item name="colorPrimaryVariant">@color/dialogLime</item>
+        <item name="colorSecondary">@color/backgroundLime</item>
+        <item name="background">@color/backgroundLime</item>
+        <item name="android:colorBackground">@color/backgroundLime</item>
+        <item name="colorOnSurface">@color/colorAccentLimeText</item>
```

Comment:
I noticed this is used for snackbar text so it shows wrong, but I'll update the attrs in other places since colorOnSurface was used in dumb places